flexget/plugins/internal/api_bluray.py

```diff
@@ -126,7 +126,10 @@ def __init__(self, title, year):
             # Used for parsing some more data, sadly with soup
             self.url = result['url']
 
-            movie_info_response = requests.get(self.url).content
+            try:
+                movie_info_response = requests.get(self.url).content
+            except (requests.RequestException, ConnectionError) as e:
+                raise PluginError("Couldn't connect to blu-ray.com. %s" % self.url)
```

Comment:
hmm, now i'm thinking maybe `PluginWarning` would be better here so we won't abort the task. @gazpachoking @cvium WDYT?

Reply:
Not a `LookupError`?